<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1619</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make Calibration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Master</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="master_date"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Output Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="master_output_directory_button">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="master_output_directory_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Sample Environment:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sample_environment_combobox">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Entry #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sample Environment</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Calibration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Output Dir.</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="remove_row_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add_row_button">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="run_calibration_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Run Calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1619</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>master_output_directory_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>master_browse_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_row_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_row_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_row_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_row_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1569</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>run_calibration_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_calibration_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>master_date</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>MainWindow</receiver>
   <slot>master_date_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>master_browse_button_clicked()</slot>
  <slot>remove_row_button_clicked()</slot>
  <slot>add_row_button_clicked()</slot>
  <slot>run_calibration_button_clicked()</slot>
  <slot>master_date_changed()</slot>
 </slots>
</ui>
